src/frontends/tensorflow/src/tf_utils.hpp

```diff
@@ -105,7 +105,7 @@ bool propagate_conditional_flow(const ov::OutputVector& ov_inputs,
 void copy_conditional_flow_marker(const CfMarkerType& copy_from, CfMarkerType& copy_to);
 
 // create Loop operation corresponding to TensorFlow While operation
-std::shared_ptr<ov::op::v5::Loop> create_loop_for_tf_while(
+std::pair<ov::OutputVector, std::shared_ptr<ov::op::v5::Loop>> create_loop_for_tf_while(
```

Comment:
please return only `ov::OutputVector`

Reply:
I described the problem with outputs in another comments in this PR. We don't know if any of loop outputs corresponds to loop.